backend/controllers/hostedByController/HostedByController.js

```diff
@@ -0,0 +1,50 @@
+const router = require('express').Router();
+const responses = require('../../constants/routeResponses');
+const HostedBy = require('../../models/HostedBy');
+const Host = require('../../models/Host');
+const CoHost = require('../../models/Cohost');
+const Language = require('../../models/Language');
+const HostLanguage = require('../../models/HostLanguage');
+
+router.get('/:propertyId', async (req, res) => {
+    try {
+        const hostedBy = await HostedBy.findOne({
+            where: { PropertyId: +req.params.propertyId },
+            include: [
```

Comment:
@manedurphy wondering if there is a more linear way to write this, it's difficult to immediate grasp the data output of all the 'include' statements